### PR TITLE
Notarise the macOS app

### PR DESCRIPTION
**Pull request by dbkr**
Notarise the mac app in the electron build process

### CODE REVIEW

File: scripts/electron_afterSign.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+exports.default = async function(context) {
+    const { electronPlatformName, appOutDir } = context;
+    if (electronPlatformName !== 'darwin') {
+        return;
+    }
+
+    // We get the password from keychain. The keychain stores
+    // user IDs too, but apparently altool can't get the user ID
+    // from the keychain, so we need to get it from the environment.
+    const userId = process.env.NOTARIZE_APPLE_ID;
```
Comment (jryans) [inline scripts/electron_afterSign.js]: Who will set this env var? Can we manually extract it from keychain with `keychain find-generic-password ...` or similar?

Comment (dbkr) [reply scripts/electron_afterSign.js]: I was just going to put it in the ~/.profile on the build machine. I'd hoped we'd be be able to just get it out of the keychain but I can't work out how to get the entry in a place in the keychain where both `altool` and `security` can find it...

**Comment by turt2live** [comment]
(@dbkr can haz a title that will look okay in the changelog please?)

File: scripts/electron_afterSign.js (hunk 1 above)
Comment (jryans) [reply scripts/electron_afterSign.js]: Aha, I see... Well, probably it's fine. 😄

Comment (martindale) [reply scripts/electron_afterSign.js]: What's the current method for getting this set up?  I've tried `NOTARIZE_CREDS=foo yarn build:electron:macos` to no avail...

Comment (jryans) [reply scripts/electron_afterSign.js]: Let's discuss in this #riot-dev.

**Pull request merged**